### PR TITLE
RFC 51: Rename lint.whitelist to lint.ignore

### PR DESCRIPTION
**Pull request by stephenmcgruer**

### CODE REVIEW

**Comment by zcorpan** [comment]
I support this change. But:

* The title says `.ignorelist` but the text says `.ignore`. Please pick one. :)
* It would be good to understand the impact of downstream projects better.

**Comment by foolip** [comment]
It's now `lint.ignore` everywhere, and I think that looks good!

File: rfcs/rename_lint_whitelist.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+[web-platform-tests.org](https://web-platform-tests.org) documentation will
+need updating.
+
+## Risks
+
+Known risks:
+
+1. Downstream consumers or infrastructure may rely on the name of our lint
+   file. The level of risk here is unknown, but not expected to be high - it
+   seems unlikely the WPT lint is a critical file.
```
Comment (stephenmcgruer) [inline rfcs/rename_lint_whitelist.md:24]:
```suggestion
1. Downstream consumers or infrastructure may rely on the name of our lint
   file. Neither Gecko and Chromium appear to rely on it, and so the risk here
   is low.
```

**Comment by stephenmcgruer** [comment]
I have updated the RFC with the note about Gecko and Chromium; hopefully that addresses @zcorpan's concerns.

**Pull request merged**